packages/ui/src/components/Stepper/index.tsx

```diff
@@ -94,102 +231,139 @@ const StyledContainer = styled.div<{
   }
 `
 
-type StepperNumbersProps = {
-  temporal: Temporal
-  children: ReactNode
-  CurrentStep: number
-  size?: 'small' | 'medium'
-}
-
-const StepperNumbers = ({
-  temporal,
+const Step = ({
```

Comment:
Can you move step into a separate file? The current file seems a bit big now, it can be nice to split in two